src/main/webapp/index.html

```diff
@@ -1,26 +1,24 @@
-<!DOCTYPE html>
 
+<!DOCTYPE html>
 <html>
-
-<head>
-	<meta charset="UTF-8">
-	<title>Givr</title>
-	<link rel="stylesheet" href="style.css">
-	<script src="index_script.js"></script>
-  <script src="user.js"></script>
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <title>Givr</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="index_script.js"></script>
+  </head>
 
 <body>
 	<div id="help-near-me-content">
 		<div id="site-title">Givr</div>
 		<div id="nav-bar">
 			<a href="index.html">Help Near Me</a>
-			<a id="login-url" href="">Login</a>
-      <a href="registration.html">Register Org</a> <!--this will only appear when logged in after login is setup -->
-			<a id="login-url">Login</a>
+            <a href = "registration.html"> Register Org</a>
+			<a href="">Login</a>
```

Comment:
Login anchor tag should have id="login-url"

Reply:
fixed